assets/javascripts/kitten/components/buttons/button.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import classNames from 'classnames'
+
+export class Button extends React.Component {
+  renderLeftIcon() {
+    if (this.props.iconOnRight) { return null }
+    return this.props.children
+  }
+
+  renderRightIcon() {
+    if (this.props.iconOnRight) { return this.props.children }
+    return null
+  }
+
+  render() {
+    const { className,
+            text,
+            children,
+            tag,
+            size,
+            iconOnRight,
+            ...rest }
+    = this.props
+
+    const buttonTypeClassName = `k-Button--${this.props.name}`
```

Comment:
On pourrait ajouter `this.props.name` dans la liste des constantes au dessus pour ne mettre que `${name}`. Qu'est-ce que tu en penses ?